optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -285,6 +286,12 @@ def _from_transformers(
         feature_extractor: Optional["CLIPFeatureExtractor"] = None,
         **kwargs,
     ):
+        import torch
+
+        if packaging.version.parse(torch.__version__) > packaging.version.parse("1.13.1") and packaging.version.parse(
+            torch.__version__
+        ) < packaging.version.parse("2.1.0"):
```

Comment:
You can also use [`is_torch_version`](https://github.com/huggingface/optimum-intel/blob/86cdea9893493c106d3be516c7a47c9acc4c8a9e/optimum/intel/utils/import_utils.py#L176):
```suggestion

        if is_torch_version(">", "1.13.1") and is_torch_version("<=", "2.0.1"):
```